src/routes/v2/authenticatedSites/media.ts

```diff
@@ -108,22 +108,20 @@ export class MediaRouter {
     { directoryName: string },
     { directories: MediaDirOutput[] },
     never,
-    { page: number },
+    never,
     { userWithSiteSessionData: UserWithSiteSessionData }
   > = async (req, res) => {
     const { userWithSiteSessionData } = res.locals
-
     const { directoryName } = req.params
-    const { page } = req.query
 
     const {
       directories,
     } = await this.mediaDirectoryService.listMediaDirectoryContent(
       userWithSiteSessionData,
       {
         directoryName,
-        page,
-        limit: 0,
+        page: 0,
+        limit: 1,
```

Comment:
May I get context behind this change?

Reply:
from huddle: we need to set this because otherwise it invokes `.value`, which will cause it to get all the files

Reply:
limit 0 loads everything as you can see in src/utils/files.ts

```
 const paginatedFiles =
    limit === 0
      ? sortedFiles.value()
      : sortedFiles
          .drop(page * limit)
          .take(limit)
          .value()
```